.yarn/patches/@metamask-eth-ledger-bridge-keyring-npm-0.13.0-189684913d.patch

```diff
@@ -0,0 +1,169 @@
+diff --git a/index.js b/index.js
```

Comment:
Ignore these patches - will be replaced with package updates. Discussions on changes are happening in their respective repos